edxval/settings.py

```diff
@@ -179,3 +179,11 @@
 
 # Set this value in the environment-specific files (e.g. local.py, production.py, test.py)
 FERNET_KEYS = ['insecure-ferent-key']
+
+# Transcript provider settings variables, which will be overridden at deployment if needed
+# NOTE: These settings must be added in edx-platform settings to allow deployment override and val integration
+CIELO24_SETTINGS = dict(
+    CIELO24_API_VERSION=1,
```

Comment:
nit: since these settings will live both on val/platform and vem, we should add a comment suggesting to change the version, whenever we do, in both places (VEM and VAL/Platform) or keep it in some config repo to update it in one place

Reply:
These settings aren't utilized by edxval as edxval is to be plugged in edx-platform. I will add the comment when adding the settings in platform.

Reply:
I added the settings here to show what set of values are present in it.

Reply:
okay